src/quacc/settings.py

```diff
@@ -158,7 +158,19 @@ class QuaccSettings(BaseSettings):
     GAUSSIAN_CMD: Path = Field(
         Path("g16"), description=("Path to the Gaussian executable.")
     )
-
+    # ---------------------------
+    # ONETEP Settings
+    # ---------------------------
+    ONETEP_CMD: Optional[Path] = Field(
+        None, description=("Path to the ONETEP executable.")
```

Comment:
What is the name of the ONETEP executable? You should use that as the default here in case it's in the environment variables.